Orange/data/io.py

```diff
@@ -440,15 +471,19 @@ def valuemap_index(val):
 
                 # Reorder discrete values to match existing variable
                 if var.is_discrete and not var.ordered:
-                    new_order, old_order = var.values, coltype_kwargs.get('values', var.values)
+                    new_order, old_order = var.values, coltype_kwargs.get(
+                        'values', var.values)
```

Comment:
You call this better, more clear? IMHO, if you wish to break on 80 so badly, at least do it properly. The example here being no pearl, but code that is worked on by lots of people is foremost meant to be readable; breaking lines at any cost does not constitute code any more readable than does breaking them poorly.